types/three/three-core.d.ts

```diff
@@ -6617,17 +6785,19 @@ export interface TextGeometryParameters {
     bevelSize?: number;
 }
 
+export class TextBufferGeometry extends ExtrudeBufferGeometry {
+    constructor(text: string, parameters?: TextGeometryParameters);
+
+    parameters: TextGeometryParameters
+
+}
+
 export class TextGeometry extends ExtrudeGeometry {
     constructor(text: string, parameters?: TextGeometryParameters);
 
     parameters: {
         font: Font;
-        size: number;
-        height: number;
-        curveSegments: number;
-        bevelEnabled: boolean;
-        bevelThickness: number;
-        bevelSize: number;
+        parameters: TextGeometryParameters;
```

Comment:
I see the paraemters of type { text: string; parameters: TextGeometryParameters } https://github.com/mrdoob/three.js/blob/ff2ce5657cf1e2567c7d240ff881b2564d133570/src/geometries/TextGeometry.js#L32